src/Core/Form/ChoiceProvider/ThemeByNameWithEmailsChoiceProvider.php

```diff
@@ -61,7 +67,7 @@ public function getChoices()
             $coreMailsFolder = $theme->getDirectory() . '/mails';
             $modulesMailFolder = $theme->getDirectory() . '/modules';
             if (is_dir($coreMailsFolder) || is_dir($modulesMailFolder)) {
-                $themeChoices[$theme->getName()] = $theme->getName();
+                $themeChoices[$theme->getName()] = realpath($this->themesDirectory . '/' . $theme->getName());
```

Comment:
I just realized something, I wonder if we even need to inject `themesDirectory` Since the Theme class already contains this folder maybe you could simply do:

```suggestion
                $themeChoices[$theme->getName()] = $theme->getDirectory();
```